akgoins/ROSRagnarEDU
# Skeleton

WIll continue to update this.

To test:

`rosrun ragnar_state_publisher ragnar_state_publisher_node`
`rosrun ragnar_kinematics ragnar_kinematics_node`

then type in an ik command and press enter, like so:

`ikp 0 0.3 -0.05 0`

This will publish a joint state command that the state publisher should pick up on and publish the relevant transforms for.
